modules/alloy/src/main/ts/ephox/alloy/dragging/dragndrop/ImageClone.ts

```diff
@@ -29,7 +27,7 @@ const createGhostClone = (image: DragnDropImageClone): SugarElement => {
 // Fire dragDrop on that image
 // remove the image in a setTimeout( 0 )
 // http://jsfiddle.net/stevendwood/akScu/21/
-const setDragImageFromClone = (transfer: DataTransfer, parent: SugarElement, image: DragnDropImageClone): void => {
+const setDragImageFromClone = (transfer: DataTransfer, parent: SugarElement<Node>, image: DragnDropImageClone): void => {
```

Comment:
Shouldn't this be parent: SugarElement<Element> or does that break to much things?

Reply:
Not sure what you mean here sorry... it's already type as `SugarElement`?

Reply:
Looking at where this is used we could make the `Node & ParentNode`, but it definitely can't be `Element` because the fallback for `getImageParent` is to get the parent node which isn't guaranteed to be an element.

Reply:
What I'm saying it should be ParentNode or even Element since it's highly unlikely that you are drag dropping the documentElement or document fragment nodes since those are not attached to anything.

Reply:
So the fallback is wrong it should be Traverse.parentElement it should all be elements.

Reply:
Have changed the function and types.